bubble-wrap-style.yaml

```diff
@@ -1382,11 +1431,111 @@ layers:
                     family: global.text_font_family
                     weight: normal
                     fill: global.text_fill
 
+#        z_debug:
+#           filter: { $geometry: line, kind: true, $zoom: { min: 1 }, source: [naturalearthdata.com, openstreetmap.org] }
+#           draw:
+#               lines:
+#                   color: |
+#                      function() {
+#                          var letters = '0123456789ABCDEF';
+#                          var color = '#';
+#                          for (var i = 0; i < 6; i++ ) {
+#                              color += letters[Math.floor(Math.random() * 16)];
+#                          }
+#                          return color;
+#                      }
+#                   order: 1000
+#                   width: 1px
+#                   visible: true
+
+        hgv-interlay:
+            filter:
+                all:
+                    - hgv: true
+                    - function() { return global.sdk_hgv_overlay; }
+            draw:
+                lines:
+                    visible: true
+                    color: blue
+                    width: 1.7px
+            z-none:
+                filter:
+                    - hgv_restriction: true
+                draw:
+                    lines:
+                        color: red
+                        width: 2.5px
+            z-restricted:
+                filter:
+                    - hgv_restriction: true
+                draw:
+                    lines:
+                        color: purple
+                        width: 2.5px
+                tunnel:
+                    filter: {is_tunnel: true, $zoom: {min: 13} }
+                    draw:
+                        lines:
+                            # color: [[16, [0.843,0.843,0.843]]]
+                            color: purple
+#                            outline:
+#                                color: [[13, [0.890,0.890,0.890]], [14, [0.870,0.870,0.870]], [16, [0.824,0.753,0.741]]]
+                z-shields:
+                    filter: { hgv_restriction_shield_text: true }
+                    draw:
+                        mapzen_icon_library:
+                            # you need to match any custom shield to the vector tile `network` values
+                            sprite: |
+                                function() {
+                                    if( feature.hgv_restriction_shield_text ) {
+                                        return ('generic_shield-' + feature.hgv_restriction_shield_text.length + 'char');
+                                    }
+                                }
+                            size: [[7,[75%,75%]],[12,[100%,100%]],[15,[110%,110%]]]
+                            priority: 1
+                            # Size is specified below, and limited to 5 char in another filter
+                            repeat_group: shields
+                            repeat_distance: [[8,90px],[10,40px],[13,30px]]
```

Comment:
I had to check in the code if `repeat_distance` supports zoom stops.. it does! Supports functions too. :) But the docs don't reflect either (updating).

@matteblair @tallytalwar does ES support zoom stops and/or functions for `repeat_distance`? Let's file an issue for parity if not.

Reply:
See: https://github.com/tangrams/tangram-es/issues/2079.